[20:58:59] [DEBUG] Running client sanity checks...
[20:59:01] [DEBUG] No setup function found in play
[20:59:01] [DEBUG] Found extension class Music in play: Attempting to load
[20:59:01] [DEBUG] Added callback: Music._menu
[20:59:01] [DEBUG] Added callback: Music._pause
[20:59:01] [DEBUG] Added callback: Music._resume
[20:59:01] [DEBUG] Added callback: Music._skip
[20:59:01] [DEBUG] Added callback: Music._stop
[20:59:01] [DEBUG] Added callback: Music.play
[20:59:01] [DEBUG] Loaded Extension: play
[20:59:01] [DEBUG] No setup function found in askbard
[20:59:01] [DEBUG] Found extension class Bard in askbard: Attempting to load
[20:59:01] [DEBUG] Added callback: Bard._askbard
[20:59:01] [DEBUG] Added callback: Bard._endbard
[20:59:01] [DEBUG] Added callback: Bard.set_bard_channel
[20:59:01] [DEBUG] Loaded Extension: askbard
[20:59:01] [DEBUG] No setup function found in noi_chu
[20:59:01] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[20:59:01] [DEBUG] Added callback: NoiChu._reset
[20:59:01] [DEBUG] Added callback: NoiChu._world_setup
[20:59:01] [DEBUG] Loaded Extension: noi_chu
[20:59:01] [DEBUG] No setup function found in db_refesh
[20:59:01] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[20:59:01] [DEBUG] Loaded Extension: db_refesh
[20:59:01] [DEBUG] No setup function found in help
[20:59:01] [DEBUG] Found extension class Help in help: Attempting to load
[20:59:01] [DEBUG] Added callback: Help.help
[20:59:01] [DEBUG] Loaded Extension: help
[20:59:01] [DEBUG] Added callback: __join
[20:59:01] [DEBUG] Added callback: __leave
[20:59:01] [DEBUG] Added callback: _about
[20:59:01] [DEBUG] Added callback: _refresh
[20:59:01] [DEBUG] Added callback: _setup
[20:59:01] [DEBUG] Added callback: on_start
[20:59:01] [DEBUG] 6 callbacks have been loaded from __main__.
[20:59:01] [DEBUG] Added callback: _disconnect
[20:59:01] [DEBUG] Added callback: _on_websocket_ready
[20:59:01] [DEBUG] Added callback: on_autocomplete_completion
[20:59:01] [DEBUG] Added callback: on_autocomplete_error
[20:59:01] [DEBUG] Added callback: on_command_completion
[20:59:01] [DEBUG] Added callback: on_command_error
[20:59:01] [DEBUG] Added callback: on_component_completion
[20:59:01] [DEBUG] Added callback: on_component_error
[20:59:01] [DEBUG] Added callback: on_error
[20:59:01] [DEBUG] Added callback: on_modal_completion
[20:59:01] [DEBUG] Added callback: on_modal_error
[20:59:01] [DEBUG] Added callback: on_resume
[20:59:01] [DEBUG] 12 callbacks have been loaded from Client.
[20:59:01] [DEBUG] Attempting to login
[20:59:02] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[20:59:02] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[20:59:02] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[20:59:03] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[20:59:03] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[20:59:03] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[20:59:03] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[20:59:03] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[20:59:03] [INFO] Shard ID 0 | Starting Shard
[20:59:03] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[20:59:03] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[20:59:03] [INFO] Shard ID 0 | Gateway connection established
[20:59:03] [DEBUG] Shard ID 0 | Session ID: 175ffc475e1e36f304cae57f53881e89 Trace: ['["gateway-prd-us-east1-d-ss4n",{"micros":143460,"calls":["id_created",{"micros":1218,"calls":[]},"session_lookup_time",{"micros":3422,"calls":[]},"session_lookup_finished",{"micros":12,"calls":[]},"discord-sessions-prd-2-183",{"micros":137332,"calls":["start_session",{"micros":68744,"calls":["discord-api-948796595-xshzp",{"micros":56199,"calls":["get_user",{"micros":10280},"get_guilds",{"micros":6637},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":2},"authorized_ip_coro",{"micros":13}]}]},"starting_guild_connect",{"micros":65,"calls":[]},"presence_started",{"micros":8741,"calls":[]},"guilds_started",{"micros":194,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":59549,"calls":[]},"connect_finished",{"micros":59556,"calls":[]},"build_ready",{"micros":27,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}]']
[20:59:03] [DEBUG] Dispatching Event: websocket_ready
[20:59:03] [DEBUG] Dispatching Event: guild_join
[20:59:03] [DEBUG] Dispatching Event: guild_join
[20:59:03] [DEBUG] Dispatching Event: guild_join
[20:59:03] [DEBUG] Dispatching Event: guild_join
[20:59:03] [DEBUG] Dispatching Event: guild_join
[20:59:04] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[20:59:04] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[20:59:04] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[20:59:04] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[20:59:04] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[20:59:04] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[20:59:04] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[20:59:04] [INFO] Overwriting 0 with 15 application commands
[20:59:05] [DEBUG] Caching ingested rate limit data for: 3e32a240d3716487d7d8f6dae6c54f6b
[20:59:05] [DEBUG] PUT /applications/{app_id}/commands Received 200 :: [1/2 calls remaining]
[20:59:05] [DEBUG] Sync of 1 scopes took 1.2120537999981025 seconds
[20:59:05] [DEBUG] Dispatching Event: startup
[20:59:05] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1702907945154, "activities": [{"name": "l\u1ec7nh /help \u0111\u1ec3 gi\u00fap \u0111\u1ee1", "type": 0}], "status": "idle", "afk": false}}
[20:59:07] [DEBUG] 1174695146214928500::play should be called
[20:59:07] [DEBUG] Dispatching Event: message_create
[20:59:07] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[20:59:07] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4NjMwNjY1NDU3NzM3MzI0NDpsT3Ywd3gxQVY2SmVOYms0aEJKQnY5bVdyS1ZJV21mbldFOGUxaWxlbnRkbjFNMkMyUDZaVlNLS0JVbmhvSWpuRnBIbmRPQUt3YjJOeXAxQnhMNHN0cFRGejVLbVNISTdhZ0Q2MEdtbER5MGtzSnlnTG9XYkFCZDRhRW5nOXVoZQ/callback Received 204 :: [4/5 calls remaining]
[20:59:07] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1186305775207989322, "self_mute": false, "self_deaf": false}}
[20:59:07] [DEBUG] Waiting for voice connection data...
[20:59:07] [DEBUG] Dispatching Event: voice_user_join
[20:59:08] [DEBUG] Attempting to initialise voice gateway...
[20:59:08] [DEBUG] Voice Connection has identified itself to Voice Gateway
[20:59:08] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[20:59:08] [DEBUG] IP Discovery in progress...
[20:59:08] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x01z\x15116.100.183.123\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xec;'
[20:59:08] [DEBUG] IP Discovered: 116.100.183.123 #60475
[20:59:08] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.183.123", "port": 60475, "mode": "xsalsa20_poly1305_lite"}}}
[20:59:08] [DEBUG] Starting UDP Keep Alive
[20:59:08] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[20:59:08] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[20:59:08] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[20:59:09] [DEBUG] Sending heartbeat every 13.75 seconds
[20:59:09] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.35137448855009956}
[20:59:09] [DEBUG] ❤ Voice Connection is sending Heartbeat
[20:59:09] [DEBUG] ❤ Heartbeat acknowledged after 0.09292 seconds
[20:59:10] [DEBUG] Sending heartbeat every 41.25 seconds
[20:59:10] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[20:59:10] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[20:59:11] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[20:59:21] [DEBUG] Caching ingested rate limit data for: 3d2712a9e4fe17cc9d3fed4a8e672e5f
[20:59:21] [DEBUG] PATCH /webhooks/{application_id}/aW50ZXJhY3Rpb246MTE4NjMwNjY1NDU3NzM3MzI0NDpsT3Ywd3gxQVY2SmVOYms0aEJKQnY5bVdyS1ZJV21mbldFOGUxaWxlbnRkbjFNMkMyUDZaVlNLS0JVbmhvSWpuRnBIbmRPQUt3YjJOeXAxQnhMNHN0cFRGejVLbVNISTdhZ0Q2MEdtbER5MGtzSnlnTG9XYkFCZDRhRW5nOXVoZQ/messages/{message_id} Received 200 :: [4/5 calls remaining]
[20:59:21] [DEBUG] Dispatching Event: command_completion
[20:59:21] [INFO] Command Called: play with event.ctx.args = ['tát nhật lãng rực rở remix fanny trần'] | event.ctx.kwargs = {'song': 'tát nhật lãng rực rở remix fanny trần'}
[20:59:21] [DEBUG] Dispatching Event: message_create
[20:59:21] [DEBUG] Caching ingested rate limit data for: 3df15bae86f6647dd4dfcbd5c6949480
[20:59:21] [DEBUG] POST /channels/1186305775207989322/messages Received 200 :: [4/5 calls remaining]
[20:59:22] [DEBUG] Dispatching Event: message_create
[20:59:22] [DEBUG] Caching ingested rate limit data for: 3df15bae86f6647dd4dfcbd5c6949480
[20:59:22] [DEBUG] POST /channels/1186305775207989322/messages Received 200 :: [3/5 calls remaining]
[20:59:22] [DEBUG] Detected ffmpeg version: 2023-11-28-git-47e214245b-full_build-www.gyan.dev
[20:59:22] [DEBUG] Now playing <YTDownloader: https://rr1---sn-8pxuuxa-8j5ek.googlevideo.com/videoplayback?expire=1702929552&ei=MFCAZej9Ibut0-kPi-eWuAI&ip=116.100.183.123&id=o-ABSxS-RB0zcziQdfQzST0V092SfSzBD1c7NzXCbmQrzp&itag=251&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=fx&mm=31%2C29&mn=sn-8pxuuxa-8j5ek%2Csn-8pxuuxa-nboer&ms=au%2Crdu&mv=m&mvi=1&pl=24&initcwndbps=3600000&spc=UWF9f56DhsbCMqH50dOOkxHDKH6WuCE&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=3907398&dur=228.761&lmt=1701812167991333&mt=1702907628&fvip=7&keepalive=yes&fexp=24007246&c=ANDROID&txp=4532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRAIgbsUcF0iaoyYZ9pvJ42kTzdf3FKf9NsAp1eepE-zxDpwCIGHvP_J30StWa75c0ZgM832BSm7nMmj2IW3Q4DZWs_Nz&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AAO5W4owRQIhANzvk6IN2evRla8679ll1Z8hhUCf6O9dZe9O2As77dT9AiAE7depWIZ85Fj5D6G6AhabFyksjGFLOlYI3WzthF3Stg%3D%3D>
[20:59:23] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.7673124881153659}
[20:59:23] [DEBUG] ❤ Voice Connection is sending Heartbeat
[20:59:23] [DEBUG] ❤ Heartbeat acknowledged after 0.09167 seconds
[20:59:37] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.5613822128011053}
[20:59:37] [DEBUG] ❤ Voice Connection is sending Heartbeat
[20:59:37] [DEBUG] ❤ Heartbeat acknowledged after 0.10403 seconds
[20:59:51] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.5532895580810852}
[20:59:51] [DEBUG] ❤ Voice Connection is sending Heartbeat
[20:59:51] [DEBUG] ❤ Heartbeat acknowledged after 0.09016 seconds
[20:59:52] [DEBUG] Sending data to websocket: {"op": 1, "d": 14}
[20:59:52] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[20:59:52] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[21:00:04] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.8183086513847438}
[21:00:04] [DEBUG] ❤ Voice Connection is sending Heartbeat
[21:00:05] [DEBUG] ❤ Heartbeat acknowledged after 0.09069 seconds
[21:00:12] [DEBUG] 1174695146214928500::play should be called
[21:00:12] [DEBUG] Dispatching Event: message_create
[21:00:12] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[21:00:12] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4NjMwNjkyNjg3NTc4NzI4NDpIbENvaHJ6S3QwMmVsSVo4em9YalpOWVN2QkwydmFyMEM0YkFrUDI3V1U2eHFDaFhjZzRlQ20xRVZNWlpaR3g2QlNwZHVoY1VOazBTYURmUWpwUDdzZFhIeUpHaWRVaU01NXd6cmNDa29SZ0FOblZRRm8zblB5R292SnQ2RTI1WQ/callback Received 204 :: [4/5 calls remaining]
[21:00:12] [DEBUG] Dispatching Event: command_error
[21:00:12] [DEBUG] Dispatching Event: command_completion
[21:00:12] [DEBUG] Dispatching Event: error
[21:00:12] [INFO] Command Called: play with event.ctx.args = ['/play song: https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': '/play song: https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[21:00:12] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 123, in play
    executor.map(lambda items: _init_music_to_queue(items), data_music[0])
                                                            ~~~~~~~~~~^^^
TypeError: 'coroutine' object is not subscriptable

[21:00:18] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.062365749063603815}
[21:00:18] [DEBUG] ❤ Voice Connection is sending Heartbeat
[21:00:18] [DEBUG] ❤ Heartbeat acknowledged after 0.09844 seconds
[21:00:19] [DEBUG] 1174695146214928500::play should be called
[21:00:19] [DEBUG] Dispatching Event: message_create
[21:00:19] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[21:00:19] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4NjMwNjk1ODIzNjU5NDI3Nzo3MG9zcDhubzZyNHNDdklnYkJVbTRBV1NxWDRoUXg5WGM0WnFCTDQ2ZnA5QmRVMHVnZDBUaGRlRHlNMzY5UlBvSlVtUVFaeTBaQlNxUVFYSzJLRmdrbklSODNiaEVTa0JqRHE0S1R3TE94MlMyQXRhWnE3SDZsd0VHSkE2WjVKRQ/callback Received 204 :: [4/5 calls remaining]
[21:00:19] [DEBUG] Dispatching Event: command_error
[21:00:19] [DEBUG] Dispatching Event: command_completion
[21:00:19] [DEBUG] Dispatching Event: error
[21:00:19] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[21:00:19] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 123, in play
    executor.map(lambda items: _init_music_to_queue(items), data_music[0])
                                                            ~~~~~~~~~~^^^
TypeError: 'coroutine' object is not subscriptable

[21:00:25] [DEBUG] 1174695146214928500::play should be called
[21:00:25] [DEBUG] Dispatching Event: message_create
[21:00:25] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[21:00:25] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4NjMwNjk4MTUxNDk3MzI4NDpSQnBzMmoxNWJEZXpaUFJ4b09jSEFXZ01QeUJPdTN2aklBNm5LRjZZa041dWR3dVJyZGdPb1VqdGVYRFB4dUxiUWdTcmV6d0RFd1NtSHEyempyNGswdlFYeXV5ZDNMNzJlSFAydmZLSm5pYzBOTW1FYUt5N0lveHF4UlVXS3h1YQ/callback Received 204 :: [4/5 calls remaining]
[21:00:25] [DEBUG] Dispatching Event: command_error
[21:00:25] [DEBUG] Dispatching Event: command_completion
[21:00:25] [DEBUG] Dispatching Event: error
[21:00:25] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[21:00:25] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 123, in play
    executor.map(lambda items: _init_music_to_queue(items), data_music[0])
                                                            ~~~~~~~~~~^^^
TypeError: 'coroutine' object is not subscriptable

[21:00:32] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.7253407434211776}
[21:00:32] [DEBUG] ❤ Voice Connection is sending Heartbeat
[21:00:32] [DEBUG] ❤ Heartbeat acknowledged after 0.09101 seconds
[21:00:33] [DEBUG] Sending data to websocket: {"op": 1, "d": 20}
[21:00:33] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[21:00:33] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[21:00:43] [DEBUG] Dispatching Event: disconnect
[21:00:43] [DEBUG] Stopping the bot.
[21:00:43] [INFO] Shard ID 0 | Stopping Shard
